racket/src/thread/sandman-struct.rkt

```diff
@@ -30,3 +31,15 @@
                  
   #:prefab)
 
+;; A `sandman` for condition wait.
+(struct cw-sandman sandman
```

Comment:
As discussed: the `cw-sandman` additions need to be part of the main `sandman` API.